src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -51,8 +51,8 @@ private static Verification VerifyOnCoreClr
 
         private void ValidateMethodImplementation_011(string source1)
         {
-            var compilation1 = CreateCompilation(source1, options: TestOptions.DebugExe,
-                                                         parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.Latest));
+            var compilation1 = CreateCompilation(source1, options: TestOptions.DebugExe, targetFramework: TargetFramework.NetStandardLatest,
```

Comment:
> Why use "latest" rather than "30"?

To make sure we are always testing against latest set of assemblies

---
In reply to: [250677330](https://github.com/dotnet/roslyn/pull/32689#discussion_r250677330) [](ancestors = 250677330)